dockerutil/image.go

```diff
@@ -22,6 +22,19 @@ import (
 
 const diskFullStorageDriver = "vfs"
 
+// Adapted from github.com/NVIDIA/nvidia-container-toolkit/internal/lookup/library.go
+// These are destination candidates for the /usr/lib directory in the container,
+// in order of priority.
+// Depending on the inner image, the desired location may vary.
+// Note that we are excluding some nvidia-specific directories here and also
+// include a fallback to /usr/lib.
+var usrLibCandidates = []string{
+	"/usr/lib/x86_64-linux-gnu",  // Debian uses a multiarch /usr/lib directory
+	"/usr/lib/aarch64-linux-gnu", // Above but for arm64.
+	"/usr/lib64",                 // Red Hat and friends.
```

Comment:
My Ubuntu system has `/usr/lib` and `/usr/lib64`, but GPUs have been working just fine on Ubuntu. IDK how to make this algorithm not change the path for existing systems though...

Reply:
It also has:
- `/lib` => `/usr/lib`
- `/lib32` => `/usr/lib32`
- `/lib64` => `/usr/lib64`
- `/libx32` => `/usr/libx32`

Reply:
I don't think we need to worry so much about the 32-bit directories.

Reply:
I agree, just wanted to point out what was on my system. I still think that we shouldn't be using `/usr/lib64` on systems that typically use `/usr/lib` though